profiles/arvind2602.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/74208761/229057236-a196aefd-6650-4c8b-b1f9-0fe76ec31731.png",
+    "issueId": 1777,
+    "name": "Arvind Gupta",
+    "username": "arvind2602"
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```